src/components/TestSuiteRun/TestRun.jsx

```diff
@@ -59,6 +68,40 @@ class TestRunBase extends React.Component {
     }
   };
 
+  handleDownloadPSBTClick = () => {
+    const { test } = this.props;
+    const interaction = test.interaction();
+    const nameBits = test.name().split(" ");
+    const body = interaction.request().toBase64();
+    const timestamp = moment().format("HHmm");
+    const filename = `${timestamp}-${nameBits[2]}-${nameBits[1][0]}.psbt`;
+    downloadFile(body, filename);
+  };
+
+  handledDownloadWalletConfigClick = () => {
```

Comment:
Another `FIXME` for this is that this doesn't need to be done here imo. The config class (or methods wherever we end up landing) should be able to handle this. Makes the views easier to read as views, and makes things easier to maintain since we can handle the logic, particularly around the text/output in a single place.